CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 2.4.1 (Under development)
 
+### App Center Crashes
+
+* **[Feature]** Support handled errors with properties via `Crashes.trackException(throwable, properties)`.
```

Comment:
Sounds like the feature is just adding properties while it's support for handled errors as a whole. Maybe just say support sending handled errors?